django_eventstream/utils.py

```diff
@@ -79,7 +82,7 @@ def publish_event(channel, event_type, data, pub_id, pub_prev_id,
 		content_filters.append('build-id')
 	else:
 		event_id = None
-	content = sse_encode_event(event_type, data, event_id=event_id, escape=bool(pub_id))
+	content = sse_encode_event(event_type, data, event_id=event_id, escape=bool(pub_id), json_encode=bool(json_encode))
```

Comment:
The `bool()` conversion is not needed as `json_encode` is intended to be a bool.

(`pub_id` is intended to be a string, and does need the conversion)